frontend/smart-kickers-game/src/components/Heatmap/Heatmap.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import HeatMap from 'react-heatmap-grid';
+import './Heatmap.css';
+import { chooseColor } from './Colors';
+import useHeatmap from '../../hooks/useHeatmap';
+
+function Heatmap() {
+  const [{ loading, error, heatmap }] = useHeatmap();
+  if (loading) return <div className="heatmap-status">Loading...</div>;
+  if (error) return <div className="heatmap-status">Error</div>;
+
+  return (
+    <div className="heatmap-parent">
+      <div className="heatmap-container">
+        <HeatMap
+          xLabels={heatmap.array}
+          yLabels={heatmap.array}
+          data={heatmap.numbersCopy}
+          cellStyle={(background, value, min, max) => ({
+            background: `${chooseColor(value)} `,
+            fontSize: '0px',
+            color: '#444',
```

Comment:
Should you set color for font-size 0? Consider removing it

Reply:
By dafault, cell values are inserted. If font-size 0 is removed, all cell values will be visible on top of the image.